frontend/yarn.lock

```diff
@@ -1046,10 +1046,10 @@
   resolved "https://registry.yarnpkg.com/@gar/promisify/-/promisify-1.1.3.tgz#555193ab2e3bb3b6adc3d551c9c030d9e860daf6"
   integrity sha512-k2Ty1JcVojjJFwrg/ThKi2ujJ7XNLYaFGNB/bWT9wGR+oSMJHMa5w+CUq6p/pVrKeNNgA7pCqEcjSnHVoqJQFw==
 
-"@guardian/consent-management-platform@^13.5.0":
-  version "13.5.0"
-  resolved "https://registry.yarnpkg.com/@guardian/consent-management-platform/-/consent-management-platform-13.5.0.tgz#61a97c823f6b18f62a6517a3374f6e7bdfca121d"
-  integrity sha512-9iz04lyonWAbUeZ7B+fNp0KbgAervsWDO3A4CE/3e+d876AQj0I3TwU7W4jpWxI4tthcA6UVk7Y6faaS8l+lFw==
+"@guardian/consent-management-platform@^13.6.1":
+  version "13.6.1"
+  resolved "https://registry.yarnpkg.com/@guardian/consent-management-platform/-/consent-management-platform-13.6.1.tgz#528ed9193b1bd96e9f84964a5a1a080de43e29c1"
+  integrity sha512-5KQ4dZrqIBKuJCSJ/TTWXZ4F0WXY9ecpMnfOfvNIU9tXF1LAeT+LNv7hfV8eC5qMJskj2yvZZUi7hlwWD+Nq/A==
```

Comment:
This change should have been included in [PR 2515](https://github.com/guardian/membership-frontend/pull/2515). I think it’s probably ok to include them it in this one, but we should let people know that that upgrade didn’t necessarily work and maybe get them to confirm everything looks ok after merge.

Reply:
yes, I found that the Snyk [security-yarn / security](https://github.com/guardian/membership-frontend/pull/2518#logs) actions are failing for the last few PRs with the error  below.That's why I included the change in this PR.

 Failed to get dependencies for /home/runner/work/membership-frontend/membership-frontend/frontend/yarn.lock
ERROR: Dependency @guardian/consent-management-platform@^13.6.1 was not found in yarn.lock. Your package.json and yarn.lock are probably out of sync. Please run "yarn install" and try again.